include/ddprof_ffi_utils.hpp

```diff
@@ -0,0 +1,22 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
+// Datadog, Inc.
+
+#pragma once
+
+extern "C" {
+#include "ddprof/ffi.h"
+}
+
+#include "string_view.h"
+
+#include <string_view>
+
+inline ddprof_ffi_CharSlice to_CharSlice(std::string_view str) {
+  return {.ptr = str.data(), .len = str.size()};
+}
+
+inline ddprof_ffi_CharSlice to_CharSlice(string_view slice) {
```

Comment:
Extracting this makes sense.
We will have to finish our transition to c++ at one point 😄